trio/_dtls.py

```diff
@@ -608,12 +645,15 @@ def challenge_for(key, address, epoch_seqno, client_hello_bits):
     return packet
 
 
-class _Queue:
-    def __init__(self, incoming_packets_buffer):
-        self.s, self.r = trio.open_memory_channel(incoming_packets_buffer)
+T = TypeVar("T")
```

Comment:
TBF should this be `_T` since (unlike other files, where this is just implied from the filename... IDK why this is different.) we're doing the whole "underscore for private" convention.